typescript/infra/scripts/utils.ts

```diff
@@ -20,6 +27,40 @@ export async function getCoreEnvironmentConfig(
   return (await import(moduleName(environment))).environment;
 }
 
+export async function getMultiProvider(
+  environment: DeployEnvironment,
+): Promise<MultiProvider<any>> {
+  return (await import(moduleName(environment))).getMultiProvider();
+}
+
+export async function getMultiProviderRemote<Networks extends ChainName>(
+  domainNames: Networks[],
+  configs: ChainMap<Networks, TransactionConfig>,
+  environment: DeployEnvironment,
+): Promise<MultiProvider<Networks>> {
+  const connections: ChainMap<
+    ChainName,
+    {
```

Comment:
`IDomainConnection`?

Reply:
Oh ya, it wasn't exported in sdk, but just did that now